Database/Classic/classicQuestDB.lua

```diff
@@ -41,6 +41,8 @@ QuestieDB.questKeys = {
     ['parentQuest'] = 25, -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
     ['rewardReputation'] = 26, --table: {{faction(int), value(int)},...}, a list of reputation rewarded upon quest completion
     ['extraObjectives'] = 27, -- table: {{spawnlist, iconFile, text, objectiveIndex (optional), {{dbReferenceType, id}, ...} (optional)},...}, a list of hidden special objectives for a quest. Similar to requiredSourceItems
+    ['requiredSpell'] = 28, -- int: quest is only available if character has this spellID
+    ['requiredSpecialization'] = 29, -- int: quest is only available if character meets the spec requirements. Use QuestieProfessions.specializationKeys for having a spec, or QuestieProfessions.professionKeys to indicate having the profession with no spec. See QuestieProfessions.lua for more info.
```

Comment:
It says `-- AUTO GENERATED FILE! DO NOT EDIT!` at the top of the file, please :sob: :pray: 

Same for the other DB files, they are changed by editing https://github.com/Muehe/sqlua and extracting a new DB with it.

In this case we can leave it I guess, will look into getting data for those fields which I can extract, which will then just overwrite these changes. But in general don't add *anything* to these files, as the changes might get lost on DB updates otherwise.